cardinal/events/tick_results.go

```diff
@@ -0,0 +1,44 @@
+package events
+
```

Comment:
This should live in cardinal imo. EventHub doesn't rely on it as a dependency. Cardinal uses it to aggregate events and receipts then it passes it to eventhub. Cardinal is the one that depends on both eventhub and TickResults, while eventHub does not depend on TickResults. 

I'll leave this as optional because it can be moved later.

Reply:
Moved!